config/kubernetes/sig-testing/teams.yaml

```diff
@@ -59,6 +59,7 @@ teams:
     members:
     - amwat
```

Comment:
Can you drop amwat, ixdy, krzyzacy?

Reply:
Should I drop them from `sig-testing-pr-reviews` and/or `test-infra-maintainers` as well?

Reply:
Let's leave those alone

Reply:
Changes have been made